samples/ipc_hardware_update/display_1_interface/display_1_capabilities.h

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license.
+// See LICENSE file in the project root for full license information.
+
+/********************************************************************
+ * This code was auto-generated from display.1 DL and shall not be
+ * modified.
+ *
+ * All concrete functions defined in this header shall be implemented
+ * and compiled together with the interface.
+ ********************************************************************/
+
+#ifndef DISPLAY_1_CAPABILITIES_H
+#define DISPLAY_1_CAPABILITIES_H
+
+#include "az_ulib_capability_api.h"
+#include "az_ulib_result.h"
+#include "display_1_model.h"
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+  static az_result display_1_cls_concrete(
+      const display_1_cls_model_in* const in,
+      display_1_cls_model_out* out);
+
+  static az_result display_1_print_concrete(
+      const display_1_print_model_in* const in,
+      display_1_print_model_out* out);
+
+  static az_result display_1_invalidate_concrete(
+      const display_1_invalidate_model_in* const in,
+      display_1_invalidate_model_out* out);
+
+  static az_result display_1_max_x_concrete(
+      const display_1_max_x_model* const in,
+      display_1_max_x_model* out);
+
+  static az_result display_1_max_y_concrete(
+      const display_1_max_y_model* const in,
+      display_1_max_y_model* out);
+
+  static const az_ulib_capability_descriptor DISPLAY_1_CAPABILITIES[] = {
```

Comment:
Why define this here and not in the source for the interfaces?

Reply:
Both `display_1_capabilities.h` and `display_1_model.h` are autogenerated from the DTDL and shall correctly identify the interface by keeping the signature. The other codes shall be created by the developers that can define their own implementation. 

Change the position of the capability in this table would break the interface signature, so it shall be in one of the autogenerated files.

Reply:
Okay. Makes sense. Is this detailed in the readme?